rewrite-java/src/main/java/org/openrewrite/java/tree/JavaTreeCoordinates.java

```diff
@@ -0,0 +1,19 @@
+package org.openrewrite.java.tree;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.With;
+import lombok.experimental.FieldDefaults;
+import org.openrewrite.internal.lang.Nullable;
+
+@Data
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class JavaTreeCoordinates {
+
+    @With
+    J tree;
+
+    @Nullable
+    @With
```

Comment:
Don't think `@With` will be necessary here.

Reply:
done